superset-frontend/src/explore/components/useExploreAdditionalActionsMenu/index.jsx

```diff
@@ -439,6 +441,22 @@ export const useExploreAdditionalActionsMenu = (
             responsive
           />
         </Menu.Item>
+        <Menu.Item key={MENU_KEYS.VIEW_DATASET_INFO}>
+          <ModalTrigger
+            triggerNode={
+              <span data-test="view-dataset-menu-item">
+                {t('View dataset info')}
+              </span>
+            }
+            modalTitle={t('View dataset info')}
```

Comment:
from the screenshot, seems modal will only show the tables used in the dataset, while the title of "dataset info" will make me expect to see more details about the dataset itself.

Reply:
Updated to "View table info"